src/server/config.rs

```diff
@@ -29,13 +31,17 @@ pub struct Config {
     // Server listening address.
     pub addr: String,
 
+    // local store path
+    pub local_store_path: String,
```

Comment:
need local path here? I think we can use same path for raft and local storage, so here no need to distinguish it.